### ISSUE TITLE
Mapbox integration issue

### ISSUE THREAD

**Issue by ryankurte** [mapbox/geojson.io#463]
Error loading mapbox tiles, tested in chrome and safari.
Appears to be due to browser same origin policy.

Error in console is:
```
XMLHttpRequest cannot load http://a.tiles.mapbox.com/v4/tmcw.map-7s15q36b.json?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6IlpIdEpjOHcifQ.Cldl4wq_T5KOgxhLvbjE-w. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://geojson.io' is therefore not allowed access.
```

Could be because mapbox only appears to support CORS over https.
https://www.mapbox.com/developers/api/

**Comment by tmcw** [comment]
@ryankurte is this problem actually on geojson.io? Somewhere else? A custom installation? This needs much more detail.

**Comment by ryankurte** [comment]
Oops, this was on http://geojson.io. No issue on a locally hosted and TLS secured version.

I have connected to http://geojson.io now and it seems to work fine from here. I tested the same issue appears if you use a locally hosted instance with plain http, but I now realise this was also from behind our company proxy which is probably the issue.

Having a secure (TLS) version of geojson.io would resolve this for us, but it seems likely the problem is our end. Sorry about the trouble.
